v0/destinations/hs/HSTransform-v2.js

```diff
@@ -50,52 +52,66 @@ const processIdentify = async (message, destination, propertyMap) => {
   const { Config } = destination;
   const traits = getFieldValueFromMessage(message, "traits");
   const mappedToDestination = get(message, MappedToDestinationKey);
+  const checkLookup = get(message, "context.hubspotOperation");
+  // build response
+  let endpoint;
+  const response = defaultRequestConfig();
+  response.method = defaultPostRequestConfig.requestMethod;
   // if mappedToDestination is set true, then add externalId to traits
-  if (mappedToDestination) {
+  if (mappedToDestination && checkLookup) {
```

Comment:
do we want to handle mapped to destination set and lookup not done?

Reply:
this ideally will always be set up if mappedToDestination is true. So keeping this check also.